health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/configs/UICustomizations.js

```
@@ -0,0 +1,193 @@
+          const onActionSelect = async (e, row) => {
+            if (e.name == "VIEW_DASHBOARD") {
+              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${11}&microplanId=${row.id}&campaignId=${
+                row.campaignDetails.id
+              }&setup-completed=true`;
+            }
+            if (e.name == "VIEW_CUSTOM_REPORT") {
+              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${11}&microplanId=${row.id}&campaignId=${
+                row.campaignDetails.id
+              }&setup-completed=true`;
+            }
+          };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)